### PR TITLE
Create a file for Rye, UK

### LINKED ISSUE #12: dabeah's recommendations for places to eat
## Overview
- [ ] create a branch
- [ ] add a file
- [ ] commit the file
- [ ] create a pull request
- [ ] collaborate
- [ ] merge

### PR DESCRIPTION
**Pull request by dabeah**
This PR closes #12

### CODE REVIEW

File: dabeah-rye.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Rye
+
+### Great Places to Eat
+
+## Fun Things to Do
```
Comment (dabeah) [inline dabeah-rye.md:5]: Wrong format

Comment (dabeah) [inline dabeah-rye.md:5]: yep

Comment (mohammed-softordi) [reply dabeah-rye.md:5]: Reveiwed by me

Comment (mohammed-softordi) [inline dabeah-rye.md:5]: A review comment

File: dabeah-rye.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+## Rye
+
+### Great Places to Eat
```
Comment (mohammed-softordi) [inline dabeah-rye.md:3]: An other review

**Review by mohammed-softordi — COMMENTED** [review]
submitted

**Pull request merged**